### DISCUSSION TITLE
OCFL backend support

### DISCUSSION THREAD

**Discussion by lnielsen** [inveniosoftware/product-rdm#154] [Ideas]

**Comment by Herrner** [comment]
This probably isn't planned for 10.0 anymore, @lnielsen ? Asking for a friend...

**Comment by lnielsen** [comment]
No, it's only in an assigned state meaning somebody is working on/off on it but no prediction about when.

OCFL is part of the Arcadia 2 project between CERN, Plazi and Data Futures

**Comment by lnielsen** [comment]
Just linking work that has been done so far:

- https://github.com/inveniosoftware/ocflcore - core python library for writing OCFL structure
- https://github.com/inveniosoftware/invenio-ocfl - generic integration of OCFL for Invenio (provides CLI command for exporting)
- https://github.com/inveniosoftware/invenio-rdm-records/tree/ocfl - integration for RDM-Records to export the records

**Comment by batnasan** [comment]
Hi @lnielsen, just curious to see if there are any updates on this?
